drivers/timers/capture.c

```diff
@@ -391,6 +391,18 @@ static int cap_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
       default:
         {
           cperr("Forwarding unrecognized cmd: %d arg: %ld\n", cmd, arg);
+          FAR long unsigned int *ptr =
```

Comment:
Please include a comment saying that the unrecognized ioctls will be forward to the Lower Half driver, ie the Pulse Counter needs this to work.